tardis/apps/publication_workflow/static/js/lib/angular-material-data-table-0.10.10/dist/md-data-table.js

```diff
@@ -0,0 +1,1486 @@
+    tElement.addClass('md-column');
+    return postLink;
+  }
+
+  function postLink(scope, element, attrs, ctrls) {
+    var headCtrl = ctrls.shift();
+    var tableCtrl = ctrls.shift();
+
+    function attachSortIcon() {
+      var sortIcon = angular.element('<md-icon md-svg-icon="arrow-up.svg">');
+
+      $compile(sortIcon.addClass('md-sort-icon').attr('ng-class', 'getDirection()'))(scope);
+
+      if(element.hasClass('md-numeric')) {
+        element.prepend(sortIcon);
+      } else {
+        element.append(sortIcon);
+      }
+    }
+
+    function detachSortIcon() {
+      Array.prototype.some.call(element.find('md-icon'), function (icon) {
+        return icon.classList.contains('md-sort-icon') && element[0].removeChild(icon);
+      });
+    }
+
+    function disableSorting() {
+      detachSortIcon();
+      element.removeClass('md-sort').off('click', setOrder);
+    }
+
+    function enableSorting() {
+      attachSortIcon();
+      element.addClass('md-sort').on('click', setOrder);
+    }
+
+    function getIndex() {
+      return Array.prototype.indexOf.call(element.parent().children(), element[0]);
+    }
+
+    function isActive() {
+      return scope.orderBy && (headCtrl.order === scope.orderBy || headCtrl.order === '-' + scope.orderBy);
+    }
+
+    function isNumeric() {
+      return attrs.mdNumeric === '' || scope.numeric;
+    }
+
+    function setOrder() {
+      scope.$applyAsync(function () {
+        if(isActive()) {
+          headCtrl.order = scope.getDirection() === 'md-asc' ? '-' + scope.orderBy : scope.orderBy;
+        } else {
+          headCtrl.order = scope.getDirection() === 'md-asc' ? scope.orderBy : '-' + scope.orderBy;
+        }
+
+        if(angular.isFunction(headCtrl.onReorder)) {
+          $mdUtil.nextTick(function () {
+            headCtrl.onReorder(headCtrl.order);
+          });
+        }
+      });
+    }
+
+    function updateColumn(index, column) {
+      tableCtrl.$$columns[index] = column;
+
+      if(column.numeric) {
+        element.addClass('md-numeric');
+      } else {
+        element.removeClass('md-numeric');
+      }
+    }
+
+    scope.getDirection = function () {
+      if(isActive()) {
+        return headCtrl.order.charAt(0) === '-' ? 'md-desc' : 'md-asc';
+      }
+
+      return attrs.mdDesc === '' || scope.$eval(attrs.mdDesc) ? 'md-desc' : 'md-asc';
+    };
+
+    scope.$watch(isActive, function (active) {
+      if(active) {
+        element.addClass('md-active');
+      } else {
+        element.removeClass('md-active');
+      }
+    });
+
+    scope.$watch(getIndex, function (index) {
+      updateColumn(index, {'numeric': isNumeric()});
+    });
+
+    scope.$watch(isNumeric, function (numeric) {
+      updateColumn(getIndex(), {'numeric': numeric});
+    });
+
+    scope.$watch('orderBy', function (orderBy) {
+      if(orderBy) {
+        if(!element.hasClass('md-sort')) {
+          enableSorting();
+        }
+      } else if(element.hasClass('md-sort')) {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
      } else if(element.hasClass("md-sort")) {
```